test/test-lib-postgresql.sh

```diff
@@ -37,4 +40,16 @@ function test_postgresql_imagestream() {
   ct_os_test_image_stream_template "${THISDIR}/imagestreams/postgresql-${OS%[0-9]*}.json" "${THISDIR}/examples/postgresql-ephemeral-template.json" postgresql "-p POSTGRESQL_VERSION=${VERSION}${tag}"
 }
 
+
+function run_latest_imagestreams_test() {
```

Comment:
This function is the same as in mysql container. Could that be rewritten so that both container images can use the same funcion?

https://github.com/sclorg/mariadb-container/blob/655aa8de41d12946b0d23c73bb06dbedb0bd3089/test/test-lib-mysql.sh#L148

Reply:
The best way would be to move database functions to container-common-scripts. 

See issue here: https://github.com/sclorg/container-common-scripts/issues/332

In case, I will have a time, I will work on it.

But Fixing tests across https://github.com/sclorg/ organization especially OpenShift 4 tests took a more time then I have expected.

Reply:
Ok.